src/LinuxTracing/PerfEvent.h

```diff
@@ -140,7 +140,7 @@ struct UprobesWithStackPerfEventData {
   uint64_t stream_id;
   pid_t pid;
   pid_t tid;
-  perf_event_sample_regs_user_sp regs;
```

Comment:
I guess we can now also delete that struct?

Reply:
No we can't because this struct is still being used by `perf_event_sp_stack_user_sample_fixed`. That's the only place it's being used though, I can make a small change to delete both structs and make due without them. `perf_event_sp_stack_user_sample_fixed` is only being used in one place only too and it will be easy to remove it.